client.go

```diff
@@ -265,6 +266,12 @@ func setupChannel(remotePort int, sshPort int, login string) int {
 	return localPort
 }
 
+func (c *SingleServerClient) CleanupDeploy(deployIdForCleanup string) error {
```

Comment:
change argument to simply be `id` (It's clear enough in context what you are talking about)